### PR TITLE
[SYCL] Add check for carrying real kernel IDs

### PR DESCRIPTION
**Pull request by denis-kabanov**

### CODE REVIEW

**Comment by denis-kabanov** [comment]
/verify with [intel/llvm#5121](https://github.com/intel/llvm/pull/5121)

**Comment by denis-kabanov** [comment]
The new test passed all checks (verified with 5121), whereas failed tests are a common problem for each new pr to llvm-test-suite.

**Comment by denis-kabanov** [comment]
@intel/llvm-reviewers-runtime, can someone review this PR?

**Review by sergey-semenov — DISMISSED** [review]
LGTM

File: SYCL/Basic/interop/check_carrying_real_kernel_IDs.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// UNSUPPORTED: cuda || hip || level_zero
```
Comment (vladimirlaz) [inline SYCL/Basic/interop/check_carrying_real_kernel_IDs.cpp]:
```suggestion
```
**REQUIRES: opencl** is enough

**Review by vladimirlaz — APPROVED** [review]

**Comment by denis-kabanov** [comment]
Jenkins/llvm-test-suite failed due to out-of-sync changes between llvm and llvm-test-suite when running /verify.

**Comment by denis-kabanov** [comment]
@intel/llvm-reviewers-runtime can we merge this?

**Pull request merged**